Source/Cogworks.Essentials/Services/MemoryCacheService.cs

```diff
@@ -83,7 +85,9 @@ public T GetOrAdd<T>(string key, Func<T> getValueFunction, int? cacheDurationInS
         /// https://michaelscodingspot.com/cache-implementations-in-csharp-net/
         public async Task<T> GetOrAddAsync<T>(string key, Func<Task<T>> getValueFunction, int? cacheDurationInSeconds = null)
```

Comment:
IMO we should rename `getValueFunction` to `factory`. What your opinion @trt @mzajkowski ?